tests/jballerina-unit-test/src/test/resources/test-src/object/anon_object.bal

```diff
@@ -292,3 +292,23 @@ function testCodeAnalyzerRunningOnAnonymousObjectsForDeprecatedFunctionAnnotatio
 isolated function Test() returns int {
```

Comment:
This isn't the correct file for this test, right?

Reply:
Changed the test